src/kudu/server/server_base.cc

```diff
@@ -128,7 +128,6 @@ DEFINE_string(user_acl, "*",
 TAG_FLAG(user_acl, stable);
 TAG_FLAG(user_acl, sensitive);
 
-#ifdef FB_DO_NOT_REMOVE
 DEFINE_string(principal, "kudu/_HOST",
```

Comment:
This is necessary for x-DC traffic. I am not sure how we handle this with replication right now. Is similar logic present in MySQL replication plugin to encrypt and send traffic? In the long term, should we aim to consolidate the logic in one place?

Reply:
brought it back.